api/all.yaml

```diff
@@ -722,6 +748,268 @@ paths:
           content:
             application/json:
               schema: {}
+  /models:
+    post:
+      operationId: createModel
+      summary: |
+        Create a model.
+        WARNING: This API is experimental and will change in future versions.
```

Comment:
It's still experimental? What planned changes do we have?

Reply:
The api is only at v0.1  This addition will roll out in v0.2 but we should keep it marked as experimental until then.

Reply:
Curious, why is `generateTemporaryModelCredentials` above not marked experimental?

Reply:
v0.1 cut was made a week ago. This change will go into v0.2